user-docs/latest/getting-started/introduction.md

```diff
@@ -1,3 +1,68 @@
+
+![Homepage of the Admin Panel](../assets/getting-started/admin-panel-homepage.png)
+
+Before going any further into this user guide, we recommend you to acknowledge the main concepts below. They will help you to understand how Strapi works, and ensure a smooth Strapi experience.
+
+- **Development environment or Production** <br> When starting working on your application, it is in development environment, which is the status for the structuration and configuration phrase of your application and its content. After deploying your application, it is in production. This status change impacts how you can use your Strapi application, as some features are only available in development environment, such as the Content-Types Builder. In the user guide, the availability or not of a feature, depending on the application status, is always mentioned in the feature's introduction.
+
+- **Versions** <br> Strapi is constantly evolving and growing. This implies that new releases are quite frequent, to improve what is already available but also to add new features to Strapi. For every new Strapi version, we communicate through our main channels and by sending notifications both on your terminal (when launching your Strapi application), and on your application's admin panel. We always recommend to use the latest version, especially since our documentation, including this user guide, are only aligned with the latest version. If you are on an older version of Strapi, some information in this documentation may not apply to your application. 
+
+- **Roles and Permissions** <br> Some features of the admin panel, as well as the content created with Strapi itself, are ruled by a system of permissions. From your Strapi application, you have the possibility to define, at a detailed level, the roles and permissions of all admin panel users and end-users. In this user guide, all features and possible options are documented. It is however possible, depending on your role and permissions, that you may not be able to access all these features and options. In that case, please refer to the main Super Admin of your Strapi application.
+
+With all this in mind, you should be ready to start your Strapi experience!
+
+
+## Accessing the admin panel
+
+The admin panel is the back office of your Strapi application. From the admin panel, you will be able to manage content types, and write their actual content. It is also from the admin panel that you will manage users, both admin panel users and end-users of your Strapi application.
+
+::: tip NOTE
+Strapi can either be used locally, or hosted on a server, making it available at a specific URL. No matter how you choose to use Strapi, your application will work the same way. The main difference is that, if using it locally, you will need to use your terminal to launch Strapi.
+:::
+
+![User profile](../assets/getting-started/login-page.png)
+
+To access the admin panel:
```

Comment:
I'm a bit confused about these steps. Having not many steps is easy to follow but I feel like, if I'm a really Strapi beginner, "how should I launch my Strapi application" and "where is this link **Open the administration**", "on which page"?

Maybe adding more precision could be a good idea or a redirection to the documentation that will explain it to you?

For the tip section, are you sure we can configure the redirection of the admin registration form?
From my knowledge of Strapi, I'm surprised of it.
But I'm sure that you can customize the `/admin` path. So I would change a bit this tip by saying that:
You can configure the path of your admin for security reasons and if you want to find back the admin URL, it's in your terminal log when you start your application.

![Screen Shot 2020-12-01 at 11 33 27](https://user-images.githubusercontent.com/5716596/100730346-3d501180-33ca-11eb-9706-a784d42c097c.png)

Reply:
While we can customize the admin panel path, we shouldn't reference that in examples. If the user has the knowledge to understand how we can just simply state "check with your admin if the path is different" or something.

Reply:
> "if I'm a really Strapi beginner, "how should I launch my Strapi application" "
Can it be our role, to invite the user to read a documentation talking about how to use a terminal? Cause actually when you know the very few commands needed, it's super easy. When you don't know them, don't know how to find them, etc. you're a bit lost. I'm not saying we should write it but in a first time, we can just find a tutorial link somewhere? I don't know just a suggestion :)

Reply:
Jim & I reviewed together this part of the documentation and decided we would keep the most generic procedure, informing administrators that as a prerequisite, their Strapi application must be launched and that they must know what is the URL to their Strapi application. A link to the installation docs that contain the command to launch Strapi should indicate that in case the prerequisites are not met, the admin should refer to the people who installed Strapi.

However, I do agree Maeva that at some point, a documentation (maybe in a more interactive format?) could be created, to explain to all our users how to launch Strapi using the terminal, even if they're non-tech users. I'll keep that in mind for future improvements 😊